tool-plugins/intellij/src/main/resources/fileTemplates/internal/Ballerina Connector.bal.ft

```diff
@@ -1,6 +1,3 @@
-#if (${PACKAGE_NAME} && ${PACKAGE_NAME} != "")
-package ${PACKAGE_NAME};
-#end
 
```

Comment:
Don't need these newlines at the beginning.

Reply:
Fixed